src/client/opamArg.ml

```diff
@@ -442,9 +454,16 @@ let create_global_options
   let debug_level = OpamStd.Option.Op.(
       debug_level >>+ fun () -> if debug then Some 1 else None
     ) in
+  let answer =
+    match yes, confirm_level with
+    | None, None -> None
+    | _ , Some c -> Some c
+    | Some true, None -> Some `all_yes
+    | Some false, None -> Some `all_no
+  in
```

Comment:
priorities over cli: confirm level > yes/no, no yes/no simultaneously

Reply:
Hmm - I think it's inconsistent that you can't give `--yes` and `--no` at the same time, yet you _can_ do `--yes --confirm-level=no`.

Overall, I think you should be able to give `--yes`, `--no` and `--confirm-level` as many times as you like - it's simply the rightmost one which wins. The rationale for that is that you can always add it in a script:

e.g.

```bash
opam_install="opam install --confirm-level=yes"
# .., many lines of bash...
# OK, risking it here...
$opam_install --confirm-level=unsafe-yes ...
```

However, it's late in the day to allow this - I might propose that for 2.2! For now, I think `--confirm-level` shouldn't be permitted with `--yes` or `--no`... what do you think?

Reply:
Ah it is not the same kind of flag (simple flag & flag with argument), it is not possible to mix them and check their order of appearance (using cmdliner api). It is only possible between yes/no. 
Will update to
* yes/no take the last in command line*
* take the confirm-level in command line
* if both defined take the confirm level value

Reply:
Sounds good - that would also be allowing `--confirm-level` to appear multiple times (and take the last one?)